core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1019,6 +1019,21 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
 //      OperandTypes.family(SqlTypeFamily.STRING, SqlTypeFamily.TIMESTAMP),
 //      SqlFunctionCategory.TIMEDATE);
 
+  @LibraryOperator(libraries = {POSTGRESQL})
+  public static final SqlFunction AGE =
+        new SqlFunction("AGE", SqlKind.OTHER_FUNCTION,
+          ReturnTypes.ARG0, null,
+          OperandTypes.family(SqlTypeFamily.DATETIME, SqlTypeFamily.DATETIME),
+          SqlFunctionCategory.TIMEDATE);
+
+  @LibraryOperator(libraries = {POSTGRESQL})
+  public static final SqlFunction REGEXP_MATCHES =
```

Comment:
we already have fucntions like REGEXP_INSTR , REGEXP_LIKE, REGEXP_SIMILAR, REGEXP_CONTAINS .... were none of these sufficient for the use case ?

Reply:
No @divyanshuS27 REGEXP_MATCHES return a set of text (ARRAY).And above functions are different return types.